docker-compose.yml

```
@@ -60,6 +60,25 @@ services:
       - webroot-network
     restart: unless-stopped
 
+  cloud-run:
+    build:
+      context: ./cloud/run
+      dockerfile: Dockerfile
+    container_name: webroot-cloud-run
+    working_dir: /app
+    volumes:
+      - ./cloud/run:/app
```

@saisandeep1701 
The main python server running at 8887 ("web" service in this docker-compose.yml) is also mounted on /app directory. If we run these 2 services together, it will lead to unclear boundaries between the 2 services.

Suggestion - lets mount cloud/run to /app/cloud_run

```suggestion
    working_dir: /app/cloud_run
    volumes:
      - ./cloud/run:/app/cloud_run
```